src/inspector/inspectorutils.cpp

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "inspectorutils.h"
+#include "qmlengineholder.h"
+
+#include <QQuickItem>
+
+// static
+QObject* InspectorUtils::findObject(const QString& name) {
```

Comment:
This is the function from the inspectorwebsocketconnection, moved in a separate file